src/internet_identity/tests/integration/http.rs

```diff
@@ -1355,6 +1355,11 @@ fn should_report_total_account_metrics() -> Result<(), CallError> {
         "internet_identity_total_application_count",
         0f64,
     );
+    assert_metric(
```

Comment:
I meant one extra test similar to `should_report_total_account_metrics`, to check that the new metric is also added when such a scenario appears.

Reply:
Oh, of course. That's true.

In that case the effort and hack is not worth the value. Ignore it.